kuka_sunrise_driver/src/kuka_sunrise/robot_manager_node.cpp

```diff
@@ -97,19 +97,14 @@ RobotManagerNode::on_configure(const rclcpp_lifecycle::State &)
 
   auto trigger_request =
     std::make_shared<std_srvs::srv::Trigger::Request>();
-  auto future_result = set_parameter_client_->async_send_request(trigger_request);
-  auto future_status = kuka_sunrise::wait_for_result(
-    future_result,
-    std::chrono::milliseconds(3000));
-  if (future_status != std::future_status::ready) {
-    RCLCPP_ERROR(get_logger(), "Future status not ready, could not set parameters");
-    return FAILURE;
-  }
-  if (!future_result.get()->success) {
-    RCLCPP_ERROR(get_logger(), "Future result not success, could not set parameters");
+
+  auto response = kuka_sunrise::sendRequest<std_srvs::srv::Trigger::Response>(
+    set_parameter_client_, trigger_request, 0, 1000);
```

Comment:
Why did you change timeout ms from 3000 to 1000? It occurs multiple times in this PR. (I mean it's ok, but I don't know whether you changed it on purpose.)

Reply:
For the changes of simple values (member variables or FRI parameters), 1000ms is more than enough, so I changed them on purpose.
Only the requests of state changes need a longer time, so I left those on the original 3000 ms